pkg/controller/node/status.go

```diff
@@ -250,23 +250,62 @@ func (ctrl *Controller) calculateStatus(fg featuregates.FeatureGate, mcs []*mcfg
 	conditions := pool.Status.Conditions
 	status.Conditions = append(status.Conditions, conditions...)
 
+	buildInProgress := false
+	buildSucceeded := false
+
+	if mosb != nil {
+		mosbState := ctrlcommon.NewMachineOSBuildState(mosb)
+		if mosbState.IsBuilding() {
+			buildInProgress = true
+			klog.Info("MOSB Building")
```

Comment:
I think we should add the name of the MOSB here to help keep track of which one these logs are for.  Same for below